### PR TITLE
expose variables of calling context to shell

### PR DESCRIPTION
**Pull request by c-holtermann**
I'd like to be able to access the variables of the calling context. That makes developing and debugging easier.

### CODE REVIEW

**Comment by c-holtermann** [comment]
I'll try to achieve this a different way.

File: gnucash/python/init.py [hunk 1]
```diff
@@ -96,17 +97,20 @@ def quit (self):
...
     banner = "Welcome to "+title+"!\n"
-    console = Console(argv = [], shelltype = shelltype, banner = [[banner, banner_style]], size = 100)
-
+   
     window = Gtk.Window(type = Gtk.WindowType.TOPLEVEL)
     window.set_position(Gtk.WindowPosition.CENTER)
     window.set_default_size(800,600)
     window.set_border_width(0)
+    
+    console = Console(argv = [], shelltype = shelltype, banner = [[banner, banner_style]],
```
Comment (c-holtermann) [inline gnucash/python/init.py:111]: Moving initialisation of console down makes it possible to access `window` in shell as well. Console itself can be found by `window.get_children()[0]`.

**Comment by c-holtermann** [comment]
I think this should do. If someone reviews this I'll squash and force push the commits.

**Comment by jralls** [comment]
@c-holtermann OK, looks good. Squash away!

**Comment by jralls** [comment]
Thanks!

**Pull request merged**